tests/converter/test_Bi_dirac/gold.wfnoj.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<qmcsystem>
+  <wavefunction name="psi0" target="e">
+    <determinantset type="MolecularOrbital" name="LCAOBSet" source="ion0" transform="yes" href="test.orbs.h5">
+      <slaterdeterminant>
+        <determinant id="updet" size="5">
+          <occupation mode="ground"/>
+          <coefficient size="216" spindataset="0"/>
+        </determinant>
+        <basisset/>
```

Comment:
I saw a dangling `<basisset/>` not sure if the issue already exists in upstream.

Reply:
looking into this now. this shouldn't be there since it is stored in the hdf5.

Reply:
fixed